### ISSUE TITLE
How can I filter nested json key in record_modifier?

### ISSUE THREAD

**Issue by DevRockstarZ** [fluent/fluent-bit#4541]
Hi, I'm going to use Elastic Search, Kibana and fluent bit on k8s.
But, i'm suffering filtering several keys.
I configured filter-kubernetes.conf file like this,
...
        Merge_Log           On
        Merge_Log_Key       log_parsed
...
I want to filter keys from log_parsed key. (nested JSON)

I tried like this,
...
    [FILTER]
        Name record_modifier
        Match *
        Remove_key $log_parsed.test_key_1
...

but it doesn't look to be good.

In conclusion, I want to filter logs like this.

*** Example ***
Before
`"log_parsed": {
     "test_key_1" : "hello",
     "test_key_2" : "world"
    },`

After
`"log_parsed": {
     "test_key_1" : "hello"
    },`

Thanks for reading this issue!

Have a nice day! 😄
